UI/SidePanel.cpp

```diff
@@ -3211,7 +3211,9 @@ void SidePanel::DoLayout() {
 
     // system name / droplist
     ul = GG::Pt(GG::X(MaxPlanetDiameter()), GG::Y0);
-    lr = ul + GG::Pt(ClientWidth() - GG::X(MaxPlanetDiameter()), name_height);
+    auto system_name_width = ClientWidth() - GG::X(MaxPlanetDiameter());
```

Comment:
Is any of that relevant in this case, where the type is X, a typedef of int which is immediately assigned a value?